util/convert.go

```diff
@@ -81,3 +81,34 @@ func Zero(in []byte) {
 		in[i] ^= in[i]
 	}
 }
+
+// TokenToPrefixLength should only be called during testing. The TokenToPrefix
+// function should be used otherwise.
+func TokenToPrefixLength(token string, length int) string {
```

Comment:
If this is only used during testing, can it be moved into that test file?  Or is it used in multiple test files?

We only want to put functions in `util` if they need to be used in multiple packages.